README.adoc

```diff
@@ -96,92 +34,88 @@ In addition, this repository structure has been designed to work with the follow
 
-    $ cd multi-topic-level-repo-template
-    $ rm .git -fr
-    
-3. Modify the global attributes in `docs/shared/attributes.adoc` for your project.
+To best understand the information above, see the example located in the repo.
 
-4. Make modifications to the first title:
-    1. Copy the entire `docs/my-title-a` directory and its contents to `docs/_<your_actual_first_title_name>_`. 
-    2. Rename the `enterprise` and `upstream-1` directories to match the names of your distros.
-    3. Define the *my-title-a* and *product-build* attributes in `master.adoc`.
-    Ensure the first title builds after the modifications and then repeat these steps for each additional title.
-    
-5. Remove any directories for distros that you have not customized for your project:
-		* Under `docs/`, delete the directories for any titles that you do not require, such as `my-title-b` if you do not have a second title. This helps to avoid confusion. Note that these template directories can always be copied from this repository in the future, if needed.
-  * Under `docs/_<titles>_/`, delete the directories for any distros that you do not require, such as `upstream-1` if you do not have an upstream. This helps to avoid confusion. Note that these template directories can always be copied from this repository in the future, if needed.
-    
-6. If you are migrating existing content to this repository template, it is recommended to put all content into the `topics` directories for the appropriate titles, even if the content *is not* shared between distros. (There are more reasons for putting content that isn't shared between distros of a title or multiple titles into the `_<title>_/topics` directory, at least at first, but they are beyond the scope of this README. For a discussion of the reasoning, which touches on content development workflow, contact mailto:dhensley@redhat.com[Silas].)
+=== Books, Chapters, User Stories and Modular Docs
 
-7. Remove this README.adoc file, or overwrite it with the details for your project! Do not leave this README as-is for your project!
+The _rhmap_ and _upstream-1_ folder posses their own version of the *master.adoc* file - this is the Book.
 
-8. TBD: Full instructions should be provided for hooking this repository up to Pantheon. In short:
-  * You must set the Content Directory for each book to <docs>/<title>
-  * 
+The Chapters are found within the Book (*master.adoc*).
+They are simply headings.
+Each Chapter will contain one to many User Stories.
 
-//// 
-Fix buildGuides.sh to work with multi-topic-level and re-include this info
+In *master.adoc*, the User Stories are formed using references to Modular Doc *.adoc* files.
+The structure of the User Story is defined with headings and the structure is based on an link:https://gitlab.cee.redhat.com/ccs-internal-documentation/Modular_Documentation_Project/tree/master/files[Assembly template].
```

Comment:
They are the template that people can use - they are a great guide and so I would consider them important. I'll figure out where best to put them in the repo and create another PR.